Cannot find module 'postcss-html/extract' Require ...

Error on new version Cannot find module 'postcss-html/extract' Require ...


Details:

Error: Cannot find module 'postcss-html/extract' Require stack: - /root/.vscode-server/extensions/stylelint.vscode-stylelint-0.87.3/dist/server.js     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)     at Function.Module._load (internal/modules/cjs/loader.js:725:27)     at Module.require (internal/modules/cjs/loader.js:952:19)     at require (internal/modules/cjs/helpers.js:88:18)     at MEe (/root/.vscode-server/extensions/stylelint.vscode-stylelint-0.87.3/dist/server.js:232:17287)     at BNt (/root/.vscode-server/extensions/stylelint.vscode-stylelint-0.87.3/dist/server.js:232:17396)     at Object.pNt (/root/.vscode-server/extensions/stylelint.vscode-stylelint-0.87.3/dist/server.js:232:19679)     at new e (/root/.vscode-server/extensions/stylelint.vscode-stylelint-0.87.3/dist/server.js:99:1222)     at /root/.vscode-server/extensions/stylelint.vscode-stylelint-0.87.3/dist/server.js:232:22867

--- a/package.json
+++ b/package.json
@@ -200,7 +200,7 @@
     "fast-diff": "^1.2.0",
     "lodash": "^4.17.20",
     "path-is-inside": "^1.0.2",
-    "postcss-syntax": "git+https://github.com/adalinesimonian/postcss-syntax.git#bundler-friendly",
+    "postcss-syntax": "https://git@github.com/adalinesimonian/postcss-syntax.git#bundler-friendly",
     "pkg-dir": "^5.0.0",
     "stylelint": "^13.13.1",
     "vscode-languageclient": "^6.2.0-next.1",

--- a/package.json
+++ b/package.json
@@ -200,7 +200,7 @@
     "fast-diff": "^1.2.0",
     "lodash": "^4.17.20",
     "path-is-inside": "^1.0.2",
-    "postcss-syntax": "adalinesimonian/postcss-syntax#bundler-friendly",
+    "postcss-syntax": "git+https://github.com/adalinesimonian/postcss-syntax.git#bundler-friendly",
     "pkg-dir": "^5.0.0",
     "stylelint": "^13.13.1",
     "vscode-languageclient": "^6.2.0-next.1",

--- a/package.json
+++ b/package.json
@@ -200,6 +200,7 @@
     "fast-diff": "^1.2.0",
     "lodash": "^4.17.20",
     "path-is-inside": "^1.0.2",
+    "postcss-syntax": "adalinesimonian/postcss-syntax#bundler-friendly",
     "pkg-dir": "^5.0.0",
     "stylelint": "^13.13.1",
     "vscode-languageclient": "^6.2.0-next.1",

